website/docs/developer/research/research-localstack.md

```diff
@@ -223,23 +240,34 @@ Some examples of CLI commands and their corresponding outputs are shown below.
         "LastUpdateStatus": "Successful",
         "PackageType": "Zip"
     }
+  ```
 #### SQS
 
-`awslocal sqs create-queue --queue-name test-queue`
-
+  ```commandline
+    awslocal sqs create-queue --queue-name test-queue 
+  ```
```

Comment:
See notes above about combining the code blocks and adding some comments for users.